src/VisualStudio/IntegrationTest/TestUtilities/InProcess/Editor_InProc.cs

```diff
@@ -133,6 +133,54 @@ public void MoveCaret(int position)
                 view.Caret.MoveTo(point);
             });
 
+        public void EncapsulateField()
+            => GetDTE().ExecuteCommand(WellKnownCommandNames.Refactor_EncapsulateField);
+
+        public void WaitForPreviewChanges()
+            => Retry(FindPreviewChangesDialog, v => v != null, TimeSpan.FromMilliseconds(100));
+
+        public bool IsPreviewChangesShowing()
+            => FindPreviewChangesDialog() != null;
+
+        private static AutomationElement FindPreviewChangesDialog()
```

Comment:
This should all be able to live out of proc by just asking the inproc version for the hwnd.